src/styles/sidebar/components/top-bar.scss

```diff
@@ -77,6 +77,9 @@
   &:hover {
     color: $gray-dark;
   }
+  &--active {
```

Comment:
I like showing this state! I might opt for a slight different color too, but that's not my sword to die on :)

Reply:
Yeah, I mentioned that in my PR description, that I'd like a better on state for this. Let's re-address when we get to the tuning of the relevant PR.